packages/mask/package.json

```diff
@@ -9,8 +9,11 @@
     "@dimensiondev/secp256k1-webcrypto": "1.0.0-20220412114204-be816df",
     "@ethersproject/abi": "^5.4.0",
     "@ethersproject/address": "^5.0.4",
+    "@ethersproject/bignumber": "^5.6.0",
```

Comment:
do we need so many different bignumber implementations?

Reply:
`@ethersproject/bignumber` is a really helpful, because BigNumber.from(any).toNumber() gives a valid JS number.
We usually use it to convert values like `{type: 'BigNumber', hex: '0x03'}` to number.

To avoid adding this package, I added the custom method for converting `{type: 'BigNumber', hex: '0x03'}` to number.
So, "@ethersproject/bignumber" is not used anymore.

Reply:
I don't mean you cannot use bignumber, but IIRC in our project, there's already bignumber library. Maybe you should use that.

Reply:
Yes, we use `bignumber.js` now.